src/Features/Core/Portable/EditAndContinue/ActiveStatementsMap.cs

```diff
@@ -141,7 +141,9 @@ private static bool TryGetUpToDateSpan(ManagedActiveStatementDebugInfo activeSta
             {
                 foreach (var region in regionsInMethod)
                 {
-                    if (region.OldSpan.Span.Contains(activeSpan) && activeStatementInfo.DocumentName == region.OldSpan.Path)
+                    if (!region.IsExceptionRegion &&
```

Comment:
We should not apply exception region span mapping to active statements. This only accidentally worked before because AS spans were ordered before exception regions in the map.